plugins/jobs/src/js/jobsToggleSchedule.js

```diff
@@ -0,0 +1,22 @@
+import { updateSchedule } from "#SRC/js/events/MetronomeClient";
+
+export default function jobsToggleSchedule(job) {
+  const [schedule] = job.schedules;
```

Comment:
I wonder how this can work 🤔 I assume job is of a graphql job type. This would mean schedules is a JobScheduleConnection, which is an object and not an array. The array would be `job.schedules.nodes`.

Reply:
It is working though. Or at least, do you mind checking out and test? maybe I just broke it last minute.

Reply:
Just checked again, I actually in another work, remember putting a console.log on schedules, and receiving a list of objects.

Reply:
Yeah, checked it, seems to work, but that makes no sense to me. I wonder if the job we are using is coming from graphql or not. If not it's okay, if it does we need to do some research here.

Reply:
Just checked it out, it's still using the job struct, but that should be okay, it's literally one line that will need to change

Reply:
Nice.